modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/Fields.tsx

```diff
@@ -65,6 +67,58 @@ interface ISaveFDSFieldsModalContentProps {
 	saveFDSFieldsURL: string;
 }
 
+interface IContentRendererProps {
+	cetRenderers?: FDSClientExtensionCellRenderer[];
+	item: IFDSField;
+	query: string;
+}
+
+const getRendererLabel = ({
+	cetRenderers = [],
+	rendererName,
+}: {
+	cetRenderers?: FDSClientExtensionCellRenderer[];
+	rendererName: string;
+}): string => {
+	const AVAILABLE_CELL_RENDERERS = [
+		...FDS_INTERNAL_CELL_RENDERERS,
+		...cetRenderers,
+	];
+
+	const renderer = AVAILABLE_CELL_RENDERERS.filter((renderer: any) => {
+		return renderer.name === rendererName || renderer.erc === rendererName;
+	})[0];
+
+	return renderer?.label || renderer?.name || rendererName;
+};
+
+const CellRendererComponent = ({
+	cetRenderers = [],
+	item,
+	query,
+}: IContentRendererProps) => {
```

Comment:
(nit) name here looks a bit too generic. Maybe `RendererLabelCellRendererComponent`?. It's a renderer of a field called renderer, so the naming is a bit awkward, but it is what it is :sweat_smile: 

Interface for it should be named `I*Props`. Alternatively, you can set types inline.